src/components/technologies/genericTechnologies/res.py

```diff
@@ -162,17 +168,18 @@ def _perform_fitting_ST(self, climate_data: pd.DataFrame):
         :param pd.Dataframe climate_data: dataframe containing climate data
         """
         # Todo: code this
-        print("Not coded yet")
+        pass
```

Comment:
ah here is the ST, we can also just delete it?

Reply:
I would prefer to keep it...

Reply:
okii